tests/test_parsers.py

```diff
@@ -173,17 +174,20 @@ def test_parse_sssom_json(self):
             f"{self.json_file} has the wrong number of mappings.",
         )
 
-    def test_piped_element_to_list(self):
```

Comment:
We should still have a test for this just on a different element (one that has "multivalued: true".